Automattic/wp-calypso
# Domain management: Register Add/Edit DNS record paths

Related to https://github.com/Automattic/wp-calypso/pull/97673
Closes https://github.com/Automattic/wp-calypso/issues/97280

## Proposed Changes

* Introduce **Add**/**Edit** DNS sub pages
* Matched with the Figma design proposal

## Why are these changes being made?

* Doman Management Revamp project

## Testing Instructions

* Go to `/domains/manage?flags=calypso/all-domain-management`
* Select a domain and open the "DNS Records" accordion card
* Click on the "Manage" button
* Add a new Record
* Check if remains on DNS Records page
* Select "Edit" option of the newly created record
* Update the record
* Check if remans on DNS Records page

<img width="1253" alt="Screenshot 2024-12-26 at 19 43 44" src="https://github.com/user-attachments/assets/96021156-b018-4689-8351-6348a9e0bf17" />

<img width="1251" alt="Screenshot 2024-12-26 at 19 44 14" src="https://github.com/user-attachments/assets/65eb8918-6f29-42fe-b253-a1c46af1f5f0" />

<img width="1254" alt="Screenshot 2024-12-26 at 19 44 21" src="https://github.com/user-attachments/assets/5b6469a9-35f8-429c-9f98-087bea7afab1" />


## Pre-merge Checklist
